src/AsyncWebView.Uno.WinUI/AsyncWebView/AsyncWebView.cs

```diff
@@ -34,15 +34,15 @@ namespace AsyncWebView;
 [TemplateVisualState(Name = VisualStates.ConnectivityError)]
 public partial class AsyncWebView : Control
 {
-	private const string WebViewPartName = "PART_WebView";
+	private const string WebView2PartName = "PART_webView2";
 
 	private static readonly Uri _blankPageUri = new Uri("about:blank");
 
 	private readonly ControlStateManager _state = new ControlStateManager();
 	private readonly DispatcherQueue _dispatcher;
 	private readonly ILogger _logger;
 
-	private _WebView _webView;
+	private _webView2 _webView2;
```

Comment:
You could have limited the changes by keeping the same field name.

Reply:
I changed it to make it clear that it's not the same control but I can change it back

Reply:
Yeah I think that would reduce the changes without any negative impacts. `_webView` is a good member name for a class named WebView2.